contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -899,11 +899,12 @@ export const Channel = new Resource({
   },
 
   softDelete(id) {
+    let modelUrl = this.modelUrl(id);
     // Call endpoint directly in case we need to navigate to new page
     return this.transaction({ mode: 'rw', source: IGNORED_SOURCE }, () => {
-      return this.table.update(id, { deleted: true }).then(() => {
-        return client.patch(this.modelUrl(id), { deleted: true });
```

Comment:
Dexie doesn't support using these kinds of promises inside of transactions, [see here](https://dexie.org/docs/Tutorial/Best-Practices#4-use-the-global-promise-within-transactions).

Reply:
This might be a useful code comment?

Reply:
I'll add it!